src/ios/removeSharedLibraries.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function removeSharedLibraries(project, libraries) {
+
+};
```

Comment:
These two functions are just dummy placeholders that we can merge to master since there's an ongoing work to implement them now. Since this PR was meant to be a base starter for them, I don't want to remove them to avoid conflicts later.